frontend/screens/BrandAccountSignupDataPreview.js

```diff
@@ -64,32 +106,107 @@ const BrandAccountSignupDataPreview = ({ route, navigation }) => {
   const renderTextRow = (label, value, isPassword) => (
     <View style={styles.rowContainer}>
       <Text style={styles.label}>{label}</Text>
-      <Text style={[styles.value, isPassword && styles.password]}>{isPassword && !showPassword ? "********" : value}</Text>
+      <Text style={[styles.value, isPassword && styles.password]}>
+        {isPassword && !showPassword ? "********" : value}
+      </Text>
       {isPassword && (
-        <TouchableOpacity style={styles.eyeIcon} onPress={() => setShowPassword(!showPassword)}>
-          <Icon name={showPassword ? "eye" : "eye-slash"} size={24} color="black" />
+        <TouchableOpacity
+          style={styles.eyeIcon}
+          onPress={() => setShowPassword(!showPassword)}
+        >
+          <Icon
+            name={showPassword ? "eye" : "eye-slash"}
+            size={24}
+            color="black"
+          />
         </TouchableOpacity>
       )}
     </View>
   );
 
+  function readImage(url, callback) {
+    var request = new XMLHttpRequest();
+    request.onload = function () {
+      var file = new FileReader();
+      file.onloadend = function () {
+        callback(file.result);
+      };
+      file.readAsDataURL(request.response);
+    };
+    request.open("GET", url);
+    request.responseType = "blob";
+    request.send();
+  }
+
+  React.useEffect(() => {
+    if (selectedAvatarIndex !== "") {
+      readImage(
+        `../assets/avatars/avatar${selectedAvatarIndex + 1}.png`,
+        function (base64) {
+          setPhoto({
+            name: `avatar${selectedAvatarIndex}`,
+            uri: base64,
+            type: "image/png",
+            isSelected: true,
+            file: `avatar${selectedAvatarIndex + 1}`,
+          });
+          setSelectedImage(base64);
+          console.log(selectedAvatarIndex, `avatar${selectedAvatarIndex + 1}`);
+        }
+      );
+    }
+  }, [selectedAvatarIndex]);
+
+  const avatarImages = [
```

Comment:
can these be kept at common place as its getting used for influencers as well right?

Reply:
Ok I will update.